tests/foreman/ui/test_template.py

```diff
@@ -512,3 +512,63 @@ def test_positive_export_template_disable_safe_render(self):
 
         :CaseImportance: Critical
         """
+
+
+class TemplateSyncTestCase(UITestCase):
+    """Implements TemplateSync tests from UI"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Setup for TemplateSync functionality
+
+        :steps:
+
+            1. Git repository must exist (in gitlab or github) and its url
+               set in ssh:// form in robottelo.constants.
+               (note: git@git... form does not work, should start with ssh://)
+            2. SSH key must be set to `foreman` user to access that git host
+               via ssh://.
+            3. Local directory /var/tmp/templatesync-{random}/ must be created.
+            4. Organization and Location must be created to isolate the
+               templates ownership.
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_templatesync_settings_is_enabled(self):
```

Comment:
`test_positive_settings_is_enabled` as you already has `TemplateSyncTestCase` class name